lib/context.js

```diff
@@ -194,6 +194,7 @@ delegate(proto, 'request')
   .access('query')
   .access('path')
   .access('url')
+  .access('accpet')
```

Comment:
Misspelled

Reply:
thx for reminding, i should have run a test